packages/FormElement/stories/examples/Everything.js

```diff
@@ -0,0 +1,124 @@
+                        <Input
+                          onChange={e => setValue(e.target.value)}
+                          value={value}
+                          hasError={Boolean(errorText)}
+                          {...a11yProps}
+                        />
+                      )}
+                    </Content>
+                    {!errorText ? <Description>These is a description of the field.</Description> : null}
+                    <Error>{errorText}</Error>
+                  </RightCol>
+                </Layout>
+              </FormElement>
+              <br />
+              <FormElement isOptional>
+                <Layout>
+                  <LeftCol width={120}>
+                    <Label help={helpContent} helpA11yText="help me">
+                      Field 2 Label
+                    </Label>
+                  </LeftCol>
+                  <RightCol>
+                    <Instructions>This are some field instructions.</Instructions>
+                    <Content>
+                      {a11yProps => (
+                        <Select
+                          placeholder="Select an option"
+                          onChange={e => setValue2(e.target.value)}
+                          value={value2}
+                          hasError={Boolean(errorText)}
+                          {...a11yProps}
+                        >
+                          <option value="Coke">Pepsi</option>
+                          <option value="Pepsi">Coke</option>
+                        </Select>
+                      )}
+                    </Content>
+                    {!errorText ? <Description>This is a description of the field.</Description> : null}
+                    <Error>{errorText}</Error>
+                  </RightCol>
+                </Layout>
+              </FormElement>
+              <br />
+              <FormElement isDisabled>
+                <Layout>
+                  <LeftCol width={120}>
+                    <Label help={helpContent} helpA11yText="help me">
+                      Field 3 Label
+                    </Label>
+                  </LeftCol>
+                  <RightCol>
+                    <Instructions>This are some field instructions.</Instructions>
+                    <Content>{a11yProps => <Textarea hasError={Boolean(errorText)} {...a11yProps} />}</Content>
+                    {!errorText ? <Description>This is a description of the field.</Description> : null}
+                    <Error>{errorText}</Error>
+                  </RightCol>
+                </Layout>
+              </FormElement>
+            </Content>
+            {!errorText ? <Description>This is a description of the fieldset.</Description> : null}
+            <Error>{errorText}</Error>
+          </RightCol>
+        </Layout>
+      </Fieldset>
+      <Rule />
+      <Button onClick={() => setErrorText("There was an error.")} size={Button.types.size.SMALL}>
+        Show Error
+      </Button>{" "}
```

Comment:
whats this " " ?

Reply:
Just a cheap + dirty way to keep the buttons from sticking together. Not how I would do it in production (I'd use CSS) but seems like the least distracting way to accomplish it.